config/features/config.go

```diff
@@ -242,11 +241,6 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		logEnabled(disableResourceManager)
 		cfg.DisableResourceManager = true
 	}
-	cfg.EnableEIP4881 = true
-	if ctx.IsSet(DisableEIP4881.Name) {
```

Comment:
You need to remove the `DisableEIP4881` and deprecate it too since it won't do anything